### PR TITLE
`atomic_flag` formatting

### PR DESCRIPTION
**Pull request by AlexGuteniev**
To have complete atomic support

### CODE REVIEW

File: include/fmt/std.h [hunk 1]
```diff
@@ -448,5 +448,18 @@ struct formatter<std::atomic<T>, Char,
   }
 };
 
+#if defined(__cpp_lib_atomic_flag_test) && __cpp_lib_atomic_flag_test >= 201907L
```
Comment (AlexGuteniev) [inline include/fmt/std.h:451]: I observe the other feature checks don't check for the feature value, shouldn't check here too?

Comment (AlexGuteniev) [reply include/fmt/std.h:451]: No, I mean make like `#ifdef __cpp_lib_optional` above without comparing the value, as we're looking for the first and (currently) the only version of the feature

Comment (vitaut) [reply include/fmt/std.h:451]: I think it's enough to check if `__cpp_lib_atomic_flag_test` is defined unless this needs to rely on some specific change to `atomic_flag` which is unlikely.

**Review by vitaut — COMMENTED** [review]
Thanks for the PR!

**Comment by vitaut** [comment]
Merged, thanks.

**Pull request merged**